tests/test_api.py

```diff
@@ -61,21 +62,15 @@ def generate_fizzbuzz_events(self):
                 # project N sends an event every Nth minute
                 if tock % p == 0:
                     events.append(
-                        enforce_table_writer(self.dataset)
-                        .get_stream_loader()
-                        .get_processor()
-                        .process_message(
+                        InsertEvent(
```

Comment:
This wasn't part of #1066 because I had written the bulk of it before #1068 and mypy was being fussy about `search_message` being in the mapping.